addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -185,7 +185,8 @@ def _l10n_eg_validate_info_address(self, partner_id, issuer=False, invoice=False
         fields = ["country_id",
                   "state_id", "city", "street",
                   "l10n_eg_building_no"]
-        if (invoice and invoice.amount_total >= invoice.company_id.l10n_eg_invoicing_threshold) or self._l10n_eg_get_partner_tax_type(partner_id, issuer) != 'P':
+        partner_type = self._l10n_eg_get_partner_tax_type(partner_id, issuer)
+        if partner_type != 'P' and invoice and invoice.amount_total >= invoice.company_id.l10n_eg_invoicing_threshold:
```

Comment:
@ThomasBeckers Why are these changes necessary?  Because the first and should be or.   I don't think we need "invoice and" because invoice is always there.

Reply:
I included this because it was in the commit BAA is working on. If you are not confident with it I can remove it. The most important for me part is the second change with "type".

Reply:
@jco-odoo this change is because individuals do not need to have a vat number. So we want to enforce the vat only for businesses that have an invoice greater than the threshold
Regarding the "invoice and", I left it there since it was in the original code. I assumed it came from a bug fix

Reply:
@bakboka I assume it is the method being called for cases where you do not have invoice.  The logic in the documentation (https://sdk.invoicing.eta.gov.eg/documents/invoice-v1-0/#receiver :  https://sdk.invoicing.eta.gov.eg/documents/invoice-v1-0/#receiver)is that you do not need the vat number if you are Person buyer and below the treshold, so you need one when you are not Person or exceed the treshold.  So, the documentation is wrong the way I read it?  @bakboka

Reply:
@jco-odoo I think the documentation might be outdated but lets wait for @nadaseder to come back from vacation to merge that particular line